packages/attention/index.js

```diff
@@ -142,7 +146,10 @@ class FabricAttention extends kebabCaseAttributes(FabricElement) {
   get _arrowClasses() {
     return classes({
       [c.arrowBase]: true,
-      [`-${this._arrowDirection}-8`]: true,
+      [`-top-8`]: this._arrowDirection === 'top',
+      [`-right-8`]: this._arrowDirection === 'right',
+      [`-bottom-8`]: this._arrowDirection === 'bottom',
+      [`-left-8`]: this._arrowDirection === 'left',
```

Comment:
Wouldn't the original solution work with the new setup?

Reply:
Unfortunately not. Tailwind purge needs to match on full strings at build time so we can’t dynamically assemble these.